### ISSUE TITLE
Color contrast fails in Game screen

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/number-pairs#371]
**Test device**
Asus Zenbook

**Operating System**
Win 11

**Browser**
Firefox

**Problem description**
From https://github.com/phetsims/qa/issues/1330

Color contrast fails in Game screen when failing
- "Try Aagain" 2.72:1
<img width="178" height="72" alt="Image" src="https://github.com/user-attachments/assets/47c49d42-26bd-4bfe-86e3-3f53d1284565" />

- Orange dashed line (yellow inside) 2.4:1
<img width="129" height="118" alt="Image" src="https://github.com/user-attachments/assets/8179a629-04d4-45a9-a1f3-db2a02367cc2" />

- Answer Button Group 1.19:1
<img width="101" height="103" alt="Image" src="https://github.com/user-attachments/assets/e21fdd14-1502-4335-92d3-0333cb14d368" />

- Answer Button Group 2.61:1
<img width="129" height="118" alt="Image" src="https://github.com/user-attachments/assets/23e08698-289b-4f00-82e6-cccc3ffeb665" />

- Gray dashed line (blue inside) 1.93:1
<img width="129" height="118" alt="Image" src="https://github.com/user-attachments/assets/6065e13e-1768-40bb-9ed3-8cdc64a795b7" />

- Challenge Reset Button (border vs background) 1.7:1
<img width="129" height="118" alt="Image" src="https://github.com/user-attachments/assets/5bc777c4-c9ca-49e9-90de-302d8bc769dc" />

- Orange dashed line (blue inside) 1.56:1
<img width="129" height="118" alt="Image" src="https://github.com/user-attachments/assets/c4aa1bc4-c142-47ad-8a97-828f687ab515" />

**Comment by catherinecarter** [comment]
You're right, the yellow definitely gets washed out. I noticed the border around each number for all 3 colors is no longer there. I wonder how the lighter yellow will do when there's a border around it?

With the border, let's try #FFFADB and see if that's better.

**Comment by marlitas** [comment]
I'm not sure I understand the goal here... I changed the value of the answerButtonLeftAddendColorProperty but it still looks dingy and dark, not at all the color I provided... It looks like there's code in `AnswerButtonAppearanceStrategy` that is making the pressed colors darker according to the interaction state. Is that what we're wanting?

<img width="509" height="490" alt="Image" src="https://github.com/user-attachments/assets/be1d2949-c990-4d7d-b103-367de586b774" />

Or we want the lighter color only to look something like this?

<img width="144" height="726" alt="Image" src="https://github.com/user-attachments/assets/692df49c-f012-4c58-9bb7-e6ab99e7396d" />

Or something that more closely matches the behavior of the radio button group in the other screens? (This messes with opacity and has a black border instead of trying to darken or lighten colors. It also reverses the color behavior with the full opacity being the selected number and the idle buttons being lighter.)

<img width="119" height="403" alt="Image" src="https://github.com/user-attachments/assets/e3b6e798-aaab-460b-aa90-5818428ac7df" />

**Comment by marlitas** [comment]
This is done above. Over to you for review @catherinecarter. Feel free to close if it looks good!
